frontend/src/actions/users.js

```diff
@@ -66,6 +66,22 @@ export function fetchMe() {
   };
 }
 
+//request to update email and nickname of user in account page
+export function updateMe(data) {
+  console.log('updateMe in user');
```

Comment:
Lets not commit log messages like this to master. It's fine for development, but don't include them in your PR. Only substantial log messages should be committed to master.